remotespark/utils/sparkevents.py

```diff
@@ -0,0 +1,32 @@
+import remotespark.utils.configuration as conf
+
+
+class SparkEvents:
+    def __init__(self):
+        """
+        Create an instance from the handler mentioned in the config file.
+        """
+        events_handler = self._import_handler(conf.events_config()["handler"])
+        self.handler = events_handler()
+
+    def _import_handler(self, name):
+        """
+        Importing the handler based on the passed parameter.
+        """
+        components = name.split('.')
+        mod = __import__(components[0])
```

Comment:
I'm reading online that `__import__` is an internal implementation detail.  Can you use `importlib` instead?